Delete_User.css

```diff
@@ -20,52 +27,62 @@ body{
 
 nav{
 	width:100%;
-	background-color: white;
+	background-color: #0C59CF;
 	overflow:auto;
 	
 }
 
 ul{
 	padding:0;
-	margin:0 0 0 150px;
+	margin:0 0 0 30px;
 	list-style: none;
 }
-
+/* Set up position for the text in nav to the right */
 li{
-	float:left;
+	float:right;
```

Comment:
I make the link in the nav bar to be in the right side. Because I released if the nav bar was all in the left side then it is look a bit empty so I will make the logo to be in the left and the link to be in the right side so the navigation bar to look more balance.